ui/webui/src/components/review/ReviewConfiguration.jsx

```diff
@@ -200,7 +200,8 @@ export const ReviewConfigurationConfirmModal = ({ idPrefix, onNext, setNextWaits
                 key="confirm"
                 onClick={() => {
                     setNextWaitsConfirmation(false);
-                    onNext();
+                    setInstallationProgress(true);
+                    window.location.hash = "#/installation-progress";
```

Comment:
Please use cockpit.location.go instead of writing to the window.location.hash object.